### PR TITLE
traildiscover schema regex

### PR DESCRIPTION
**Pull request by arielkr256**
### Background

Adds schema regex for TrailDiscover managed schema.

### Changes

* Updates schema_regex.py with regex string for TrailDiscover schema
* https://github.com/panther-labs/panther-analysis/pull/1177
* https://github.com/panther-labs/panther-enterprise/pull/18226

### Testing

* make test

### CODE REVIEW

**Review by le4ker — APPROVED** [review]

**Review by nskobov — COMMENTED** [review]
[the schema in panther should exist first prior to merging this](https://github.com/panther-labs/panther-enterprise/pull/18226#discussion_r1546215649). lets hold off until its merged in the backend

**Comment by arielkr256** [comment]
panther-enterprise dependency has been merged https://github.com/panther-labs/panther-enterprise/pull/18226

**Review by le4ker — APPROVED** [review]

**Pull request merged**
